### ISSUE TITLE
Modal heading should be ‘h2’ (DAC usability issue)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2335]
DAC: Modal heading should be ‘h2’

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/discussions/1771#discussioncomment-3180926_

Page 47 of EQ RAS DAC report July 2022...

-----

Headings 3 (usability)
The modal heading may not be at the correct level heading.
WCAG Reference:
Usability
Issue ID: DAC_Headings3_usability
URL: https://test-runner.eq.gcp.onsdigital.uk/questionnaire/timeout-modal-interstitial/
Journey: Task 2 step 2

The modal does not appear to be implemented within the live service as of yet. When it is part of the live service the heading that introduces the modal content should be a h2 (if there is already a h1 on the page behind)

Current code ref(s): #ons-modal-title 
```
<h1 id="ons-modal-title" class="ons-modal__title"> You will be signed out soon </h1>
```

Screen reader comments: “The timeout modal was displayed within the expected time, but I observed that it was presented with a level 1 heading, whereas I would have expected it to be at level 2. The issue is present with NVDA, JAWS, VoiceOver, TalkBack.”

Solution:
When the timeout modal is implemented within the live service then the modal heading should be a h2.
